sdks/python/apache_beam/coders/coders.py

```diff
@@ -1014,6 +1016,42 @@ def as_deterministic_coder(self, step_label, error_message=None):
     return self
 
 
+class ProtoPlusCoder(FastCoder):
+  """A Coder for Google Protocol Buffers wrapped using the proto-plus library.
+
+  ProtoPlusCoder is registered in the global CoderRegistry as the default coder
+  for any proto.Message object.
+  """
+  def __init__(self, proto_plus_message_type):
+    # type: (Type[proto.Message]) -> None
+    self.proto_plus_message_type = proto_plus_message_type
+
+  def _create_impl(self):
+    return coder_impl.ProtoPlusCoderImpl(self.proto_plus_message_type)
+
+  def is_deterministic(self):
+    return True
```

Comment:
Is this deterministic ( looks like ProtoCoder is not deterministic) ?

Reply:
Yes- it uses the deterministic proto coder